### ISSUE TITLE
Provide Mono builds

### ISSUE THREAD

**Issue by Cretezy** [Zylann/godot_voxel#244]
It would be incredibly useful if the project offered Mono builds as well as standard builds for it's pre-compiled binaries.

I have tried building my own Godot but I can't get the Mono build to build properly locally (msbuild failure).

**Comment by Zylann** [comment]
I have never made Mono builds before, but if you can find a way to add it to the Github Actions CI, you could do a PR. Otherwise I can try as well but it will take some time

**Comment by Cretezy** [comment]
https://docs.godotengine.org/en/stable/development/compiling/compiling_with_mono.html

Compiling with Mono is an additional ~2 steps or so, and should only require dotnet/msbuild as an extra dependency

**Comment by Cretezy** [comment]
I was able to build Godot with this module (and Mono) successfully. What I was doing wrong was using mismatched version (I had used the 3.2.3 tag on Godot's repo and the 3.2.3 branch from this) when I needed to use the 3.2 branch of Godot and the master branch of this repository.

Would be useful to have improved documentation on this front, specifically regarding the Godot<->Voxel Tool version match (as it's not _super_ clear in the documentation). With the proper info, I could open a PR for this.
